### PR TITLE
PP-2982 Publish metric to hosted graphite with connector build time [DO NOT MERGE]

### PR DESCRIPTION
**Pull request by juanjoqmelian**
## WHAT
We want to introduce some metrics to measure the time that it takes to build connector and run its tests. In that way we will be able to measure the improvement after the work is done to only spin postgres docker container once (before all tests start).

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -28,9 +28,34 @@ pipeline {
...
+                    exit 1
+                fi
+
+                if [ -z "${HOSTED_GRAPHITE_ACCOUNT_ID}" ] ; then
+                    echo "You must supply an account id for the carbon prefix"
+                    exit 1
+                fi
+
+                if [ "${HOSTED_GRAPHITE_API_KEY}" ]; then
+                  METRIC_NAME="${HOSTED_GRAPHITE_API_KEY}.$1"
```
Comment (mrlumbu) [inline Jenkinsfile]: should this be `HOSTED_GRAPHITE_ACCOUNT_ID ` not `HOSTED_GRAPHITE_API_KEY`?

File: Jenkinsfile [hunk 2]
```diff
@@ -28,9 +28,34 @@ pipeline {
...
+      sh (
+        script: """
+            function postMetricToGraphite() {
+
+                if [ $# -ne 2 ]; then
+                    echo "You must supply metric name and metric value"
+                    exit 1
+                fi
+
+                if [ -z "${HOSTED_GRAPHITE_ACCOUNT_ID}" ] ; then
```
Comment (mrlumbu) [inline Jenkinsfile]: is this set on jenkins then?

Comment (juanjoqmelian) [reply Jenkinsfile]: yes, merged PR is here:

https://github.com/alphagov/pay-chef/pull/368

File: Jenkinsfile (hunk 1 above)
Comment (juanjoqmelian) [reply Jenkinsfile]: I haven't checked on hosted graphite docs, but that's what we have in other places where we send metrics. So I assume that is the api key what has to be sent.

File: Jenkinsfile (hunk 2 above)
Comment (mrlumbu) [reply Jenkinsfile]: 👍

File: Jenkinsfile (hunk 1 above)
Comment (mrlumbu) [reply Jenkinsfile]: ok

**Pull request closed without merging**